TowersOfHanoi/src/codeunit/Codeunit50000.SolveTowersOfHanoi.al

```diff
@@ -0,0 +1,48 @@
+codeunit 50000 SolveTowersOfHanoi
+{
+    trigger OnRun()
+    begin
+    end;
+
+    var
+        Err: Label 'Maximum Tower size is %1';

```

Comment:
Comment to explain %1

Reply:
Labels also needs to have suffix Lbl or something